kalliope/core/ConfigurationManager/BrainLoader.py

```diff
@@ -22,8 +25,9 @@ class BrainLoader(object):
     __metaclass__ = Singleton
 
     def __init__(self, file_path=None):
-        logger.debug("Loading brain with file path: %s" % file_path)
         self.file_path = file_path
+        if self.file_path is None:
```

Comment:
Basically so should reuse the same method you have done in triggerModule.py to get your file_path